src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -77,10 +80,37 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
           ) : (
             <>
               <FontAwesomeIcon icon={faChevronLeft} size="sm" />
-              <FontAwesomeIcon icon={faCommentDots} size="lg" />
+              <div className="chat-sidebar__controller-wrapper">
+                <FontAwesomeIcon icon={faCommentDots} size="lg" />
+                <span className="chat-sidebar__controller-text">Chat</span>
+              </div>
             </>
           )}
         </button>
+        {!isExpanded && (
+          <button
+            aria-label={isExpanded ? "Hide chat" : "Show chat"}
+            className="chat-sidebar__controller chat-sidebar__private-chat"
+            onClick={togglePrivateChatSidebar}
+          >
+            {isExpanded ? (
+              <FontAwesomeIcon icon={faChevronRight} size="sm" />
+            ) : (
+              <>
+                <FontAwesomeIcon icon={faChevronLeft} size="sm" />
+                <div className="chat-sidebar__controller-wrapper">
+                  <FontAwesomeIcon icon={faEnvelope} size="lg" />
+                  <span className="chat-sidebar__controller-text">
+                    Messages
+                  </span>
+                  {newPrivateMessageRecived > 0 && (
+                    <div className="chat-sidebar__controller--new-message"></div>
+                  )}
```

Comment:
is it possible to highlight the same way the public "Chat" if you have unread messages there? or we don't have such information at the moment?

Reply:
made a reference to this in https://github.com/sparkletown/internal-sparkle-issues/issues/1271#issuecomment-934281033